test/httpboot/httpboot_api_test.rb

```diff
@@ -0,0 +1,68 @@
+require 'test_helper'
+require 'tempfile'
+require 'httpboot/httpboot_plugin'
+require 'httpboot/httpboot_api'
+
+ENV['RACK_ENV'] = 'test'
```

Comment:
It feels weird to set an env var here. Is this needed?

Reply:
Copy pasted from other file, thought it's required.